app/javascript/dashboard/i18n/locale/en/chatlist.json

```
@@ -50,6 +50,9 @@
       },
       "sort_on_priority": {
         "TEXT": "Priority"
+      },
+      "sort_on_waiting_since": {
+        "TEXT": "Longest waiting"
```

```suggestion
        "TEXT": "Oldest Unanswered"
```

A suggestion (feel free to ignore it) We can perhaps use a better copy. Longest waiting may not be very clear.

---

Same suggestion. What about the text `Outstanding Replies`?

---

"pending response" ? :D

---

Let us go with "pending response" for now.